LibertyDSNP/frequency
# Bug: Fix api augment module

# Goal
The goal of this PR is to update frequency-chain/api-augment to work in an esm package.

Part of #1116

# Discussion

- Updated depedencies
- Updated default node to v18
- Fix `make js` to always `npm pack`
- Clean up and apply a simple `package.json` to the esm and cjs folders with the right `type`

## Testing
- It's a pain.
- Best to wait until this merges and we cut 1.5.2
- You really want to?
- Ok.
- run make integration-test
- That should work (common-js usage)
- `touch ./js/api-augment/dist/yarn.lock`
- Switch to a folder with the `polkadotjs/apps` repo
- Search the repo for `https://github.com/polkadot-js/apps/pull/9243` and uncomment the lines about frequency
- `yarn`
-`yarn build` Should FAIL
- `yarn link ../frequency/js/api-augment/dist` (with your own path to frequency)
- Might need to move the change from the root package.json to the packages/apps-config package.json
- `yarn build` Should SUCCESS!
